vdbench.py

```diff
@@ -1,367 +1,367 @@
-    def run(self,vd_name, workload, diskindex):
-        '''
-        This method execute workload of VdBench tool.
-        Arguments (str, str, int): vd_name, workload,diskindex
-        Return: None
-        '''
-        workload_path, result_path = self.create_file('4-4k-4-fill.vdb', vd_name,diskindex, workload)
-        self.run_workload(workload_path, result_path,vd_name,workload)
-        if workload.strip() == 'VDI':
-            workload_path, result_path = self.create_file('vdi_fill.vdb', vd_name,diskindex,workload)
-        elif workload.strip() == 'VSI':
-            workload_path, result_path = self.create_file('vsi_fill.vdb', vd_name,diskindex,workload)
-        elif workload.strip() == 'ORACLE':
-            workload_path, result_path = self.create_file('oracle_fill.vdb', vd_name,diskindex,workload)
-        else:
-            workload_path, result_path = self.create_file('sql_fill.vdb', vd_name,diskindex,workload)
-        self.run_workload(workload_path, result_path,vd_name, workload)
-
-    def run_workload(self, workload_path, result_path, vd_name, workload):
-        '''
-        This method run workloads and stores the result
-        Arguments (str, str, str, str): workload_path, result_path, vd_name, workload
-        Return: None
-        '''
-        # Config\VdBench_config\Workload
-        path = 'cd ' + self.vdbench_path +'\n'
-        str_ = 'vdbench -f "' + workload_path + '" -o "'+result_path+'"'+"\n"
-        ssh = subprocess.Popen(["cmd"],
-                            stdin =subprocess.PIPE,
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.PIPE,
-                            universal_newlines=True,
-                            bufsize=0)
-        # Send ssh commands to stdin
-        ssh.stdin.write(path)
-        ssh.stdin.write(str_)
-        out, err = ssh.communicate()
-        print(out,err)
-        ssh.wait()
-        if 'Vdbench execution completed successfully' in out:
-            if '4-4k-4-fill' not in result_path:
-                msg = workload.upper() + ' workload executed sucessfully'
-                LogCreat().logger_info.info(msg)
-                new_path = os.path.join(result_path, "flatfile.html")
-                ResultCreation().read_result(new_path, vd_name, workload, self.new_)
-            else:
-                msg = '4-4k-4-fill workload executed sucessfully'
-                LogCreat().logger_info.info(msg)
-    def create_file(self,file_name,vd_name,diskindex,workload):
-        '''
-        This method create dynamic folder structure to store
-        results of VdBench
-        Arguments (str, str, int): file_name,vd_name,diskindex
-        Return (str, str): workload_path, result_path
-        '''
-        uri = "servers"
-        res = ILDC().do_ssy_details(uri, header=None)
-        self.build = res.json()[0]['ProductBuild']
-        workload_path = os.path.abspath(r'../../../Config/VdBench_config/Workload')
```

Comment:
Avoid hard coding of paths. Create global variables to manage

Reply:
According to PEP8 in python using global variable is bad practice.
And it is relative part it should like this only.